packages/performance/src/services/cc_service.test.ts

```diff
@@ -45,46 +45,62 @@ describe.only('Firebase Performance > cc_service', () => {
 
     afterEach(() => {
       fetchStub.restore();
+      clock.restore();
     });
 
+    // First setTimeout in cc_service#processQueue(timeOffset) happens before useFakeTimers() is
+    // called. Therefore all tests need to wait for first timeout to finish before starting the
+    // actual test logic.
+    function waitForFirstTimeout(callback: () => void): void {
+      setTimeout(() => {
+        callback();
+      }, 10 * DEFAULT_SEND_INTERVAL_MS);
+    }
+
     it('throws an error when logging an empty message', () => {
-      const logger = new Logger('@firebase/performance/cc');
-      expect(() => {
-        testCCHandler(logger, LogLevel.SILENT, '');
-      }).to.throw;
+      waitForFirstTimeout(() => {
```

Comment:
Thank you Christina for the detailed feedback and tried with the testing! I have updated the logic in `cc_service.ts` to provide an init function and updated tests to include fake timer inside `describe()`. Since this test update is separated from feature branch merge in this PR (`fireperf-update-log-format`), I created master branch PR for this https://github.com/firebase/firebase-js-sdk/pull/2506. Thank you!